### PR TITLE
Reword exporters intro to not imply the collector is a viz+analysis backend

### PR DESCRIPTION
**Pull request by cartermp**
fixes https://github.com/open-telemetry/opentelemetry.io/issues/3951

---

**Preview**: **Preview**: https://deploy-preview-3963--opentelemetry.netlify.app/docs/languages/java/exporters/

### CODE REVIEW

File: layouts/shortcodes/docs/languages/exporters/intro.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
-To visualize and analyze telemetry, export it to the
-[OpenTelemetry Collector](/docs/collector/), to a backend such as
+To verify your telemetry exports correctly, one of the best things to do is send
+it to an [OpenTelemetry Collector](/docs/collector/). It is also generally a
+best practice to use the Collector in production. If you wisht to visualize your
+telemetry, you can also export it to a backend such as
```
Comment (chalin) [inline layouts/shortcodes/docs/languages/exporters/intro.md:4]:
```suggestion
To ensure that your telemetry is exporting correctly, send
it to the [OpenTelemetry Collector](/docs/collector/). It is also considered a
best practice to use the Collector in production. To visualize your
telemetry, export it to a backend such as
```

**Review by chalin — COMMENTED** [review]
Copyedits

File: layouts/shortcodes/docs/languages/exporters/intro.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
-To visualize and analyze telemetry, export it to the
-[OpenTelemetry Collector](/docs/collector/), to a backend such as
-[Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
-[Prometheus](https://prometheus.io/), or a
+To ensure that your telemetry is exporting correctly, send it to the
+[OpenTelemetry Collector](/docs/collector/). It is also considered a best
+practice to use the Collector in production. To visualize your telemetry, export
+it to a backend such as [Jaeger](https://jaegertracing.io/),
+[Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a
```
Comment (theletterf) [inline layouts/shortcodes/docs/languages/exporters/intro.md:5]:
```suggestion
Send telemetry to the [OpenTelemetry Collector](/docs/collector/) to
make sure it's exported correctly. Using the Collector in production
environments is a best practice. To visualize your telemetry, export
it to a backend such as [Jaeger](https://jaegertracing.io/),
[Zipkin](https://zipkin.io/), [Prometheus](https://prometheus.io/), or a
```

**Review by theletterf — COMMENTED** [review]
Just some copy suggestions.

File: layouts/shortcodes/docs/languages/exporters/intro.md (hunk 2 above)
Comment (chalin) [reply layouts/shortcodes/docs/languages/exporters/intro.md]: Thank you @theletterf, that's much better! 🙏 
(I wasn't pleased with the text, but didn't have the inspiration to improve it.)

**Review by chalin — APPROVED** [review]
Works for me! ✨

**Pull request merged**